ROADMAP.md

```diff
@@ -74,12 +74,13 @@ Using `gfx-rs` for backend abstraction allows us to provide compatibility with m
 **Next steps**
  * Raw input support (winit?)
  * Gamepad support (winit?)
- * Input Mapping
+ * Input Mapping: Map input events (e.g key press) to abstract action events (`Jump`, `Forward`, ..)
```

Comment:
Serializing with serde seems like the correct approach. In an file we specify something like this
```toml
# TOML syntax in config.toml, I am not familiar with yaml syntax, but should be the same.
[key-map]
up = "w" # all of these should be parsed as lowercase, so case shouldn't matter
down = "s"
left = "a"
right = "d"
run = "shift"
config = "^m" # `^` is CTRL, parsing should support "ctrl+m" aswell.
jump = "space" # is ` ` same as spacebar?
```
I agree we should open an issue specifying the specifics